data_service/adapters/storage/local.py

```diff
@@ -20,7 +20,7 @@ def get_parquet_file_path(self, data_structure_name: str, version: str) -> str:
         version_underscored = version.replace('.', '_')[:5]
         path_prefix = f"{self.settings.DATASTORE_DIR}/data/{data_structure_name}"
 
-        if version == "0.0.0":
+        if version.startswith("0.0.0") or version == "draft":
             parquet_file_path = f"{data_structure_name}__{version_underscored[:3]}"
```

Comment:
This will still look for data_version files with only 2 version values ☝🏻

Reply:
never mind this is the parquet file path 🤦🏻 my bad

Reply:
It was actually a good point, but it will fail on line 20 🤕

Reply:
Ah, true! An unintentional good point 😅 I'll take the win 🏅